check_net.sh

```diff
@@ -11,7 +11,7 @@ while true;
 do
 	date="[$(date +'%b %d, %r')] ";
 	works='true';
-	for check_address in "${address[@]}"
+	for check_address in "${ip_addresses[@]}"
 	do
 		if ping -c 1 $check_address > /dev/null;
```

Comment:
> I thought 1 second is not really effective way of testing it...

hmm... why not? 1000 milliseconds for an IP not to answer is too much...

Idea is to ping 3 IPs, each ping count (-c) is 1 and timeout 1 sec. 
If all 3 IPs do not work, then make an alert and write it down...

I think it's legit situation to call ISP